src/chains/ethereum/ethereum/tests/api/eth/getProof.test.ts

```diff
@@ -0,0 +1,258 @@
+import getProvider from "../../helpers/getProvider";
+import assert from "assert";
+import { EthereumProvider } from "../../../src/provider";
+import compile, { CompileOutput } from "../../helpers/compile";
+import { join } from "path";
+import { Data, keccak, Quantity } from "@ganache/utils";
+import { KECCAK256_NULL } from "ethereumjs-util";
+import { SecureTrie } from "merkle-patricia-tree";
+import { Account } from "@ganache/ethereum-utils";
+import { Address } from "@ganache/ethereum-address";
+
+describe("api", () => {
+  describe("eth", () => {
+    describe.only("getProof", () => {
+      let provider: EthereumProvider;
+      let contract: CompileOutput;
+      let ownerAddress: string;
+      let ownerBalance: string;
+      let contractAccount: Account;
+      const expectedContractAccountProof = [
+        "0xf901d1a0bddaa54ff11e3d79d1aa0f9df7ed9a8f9afbc0cdd35183106c0c3c377fd587cba0ab8cdb808c8303bb61fb48e276217be9770fa83ecf3f90f2234d558885f5abf180a0cb5392b1cf13d4255ffc62eec25be303d77b795bdb0e21835ded94d29323342ea0de26cb1b4fd99c4d3ed75d4a67931e3c252605c7d68e0148d5327f341bfd5283a05f1672e7a13fc3d588c018f066a010bbc3c2171c0435e17af22e2429fd868917a0c2c799b60a0cd6acd42c1015512872e86c186bcf196e85061e76842f3b7cf86080a02e0d86c3befd177f574a20ac63804532889077e955320c9361cd10b7cc6f5809a066cd3ba8af40fe37d4bfa45f61adb46466d589b337893028157f280ecc4d94f0a060ba1f8a43e38893005830b89ec3c4b560575461c3925d183e15aed75f8c6e8fa0bca2657fd15237f0fdc85c3c0739d8d7106530921b368ca73c0df81d51bcadf4a029087b3ba8c5129e161e2cb956640f4d8e31a35f3f133c19a1044993def98b61a06456f0a93d16a9f77ff0d31cf56ef090d81c2c56b12535a27c2c7c036dc8186da0a390f135abc61e0c4587b388cf0ba75d5858a1b35511d9e059c42baecb00635ea0144540d36e30b250d25bd5c34d819538742dc54c2017c4eb1fabb8e45f72759180",
+        "0xf869a03b2c95fd2a6e48d2dcb37be554d03b55e31ec582b450211db4e9c3883ffac678b846f8440180a0fe26ce2b5dafd303bdbd9adbd1bbc1ef59fceea4f81c282158ab624e2fd1787da067b59c91b38bdcd65d2f8129925a939a1469b88b66ef65395d22f74476582a5f"
+      ];
+      const emptyMerkelRoot = Data.toString(new SecureTrie().root);
```

Comment:
you spelled merkle wrong

Reply:
So it's _not_ named after https://en.wikipedia.org/wiki/Angela_Merkel???